File: Source/app/src/main/java/nl/endran/locust/fragments/ResourcesFragment.java
/*
 * Copyright (c) 2015 by David Hardy. Licensed under the Apache License, Version 2.0.
 */

package nl.endran.locust.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import nl.endran.locust.R;

public class ResourcesFragment extends Fragment {

    public static ResourcesFragment createInstance(){
        return new ResourcesFragment();
    }

    @Nullable
    @Override
    public View onCreateView(final LayoutInflater inflater, final ViewGroup container, final Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_resources, container, false);
    }
}

File: Source/app/src/main/java/nl/endran/locust/fragments/ResourcesFragment.kt
/*
 * Copyright (c) 2015 by David Hardy. Licensed under the Apache License, Version 2.0.
 */

package nl.endran.locust.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import nl.endran.locust.R

class ResourcesFragment : Fragment() {

    companion object Factory {
        fun createInstance(): ResourcesFragment = ResourcesFragment()
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.fragment_resources, container, false)
    }
}


// Instar
// Pupa

// These are stages the egg transform into before coming out of the POP, then becomung a Nymph